samples/nrf9160/fmfu_smp_svr/prj.conf

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Some command handlers require a large stack.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Ensure an MCUboot-compatible binary is generated.
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable the UART mcumgr transports.
+CONFIG_MCUMGR_SMP_UART=y
+CONFIG_UART_MCUMGR_RX_BUF_SIZE=512
+CONFIG_MCUMGR_BUF_SIZE=512
+
+# Required by the `taskstat` command.
+CONFIG_THREAD_MONITOR=y
+
+# Enable statistics and statistic names.
+CONFIG_STATS=y
+CONFIG_STATS_NAMES=y
+
+# Enable STAT and OS from the core commands
+CONFIG_MGMT_FMFU=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_STAT_MGMT=y
+
+CONFIG_LIBMODEM=y
+CONFIG_LIBMODEM_SYS_INIT=y
+CONFIG_NRF_MODEM_BUILD_FROM_SRC=y
+CONFIG_AT_CMD=y
+CONFIG_AT_CMD_PARSER=y
+CONFIG_AT_CMD_SYS_INIT=y
+CONFIG_MODEM_INFO=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y 
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+# RTT Logging since you cannot use the UART while transfering since it is used 
+# for SMP
+#CONFIG_USE_SEGGER_RTT=y
```

Comment:
get rid of these, put them in a kconfig fragment (also ((incorrectly)) known as 'overlay') instead